src/pages/api/webhooks/webhook.ts

```diff
@@ -0,0 +1,39 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+interface IssuePayload {
+  issue: {
+    html_url: string;
```

Comment:
I haven't coded much in typescript, but are names supposed to be camel cased or snake case?

Reply:
camel case, but a lot of people people send JSON as snake_case and so the lines blur because JSON <-> JS are 1:1. I think for networking-related objects, snake case is fine